### PR TITLE
[W5.11][T16-2] Lee Ming Liang

### PR DESCRIPTION
**Pull request by IcedCoffeeBoy**
Added Sort command: 
- Created a new Sortcommand class 
- Implement Comparable to person class 
- Updated the user guide

### CODE REVIEW

**Comment by IcedCoffeeBoy** [comment]
Added new test cases

**Comment by Jeevz10** [comment]
Looks good!

**Comment by r-farkhutdinov** [comment]
Good job!

**Comment by perrythewang** [comment]
Works great!

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -22,11 +20,11 @@
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
         String trimmedAddress = address.trim();
-        this.isPrivate = isPrivate;
+        super.isPrivate = isPrivate;
```
Comment (stephlewyh) [inline src/seedu/addressbook/data/person/Address.java:23]: why do you replace `this` with `super`. is it your intention to set `isPrivate` in your superclass but not for
the address class? this refactoring does not impact your enhancement directly.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (stephlewyh) [inline src/seedu/addressbook/parser/Parser.java:14]: Please state your libraries explictly.

**Review by stephlewyh — COMMENTED** [review]
Hi Ming Liang, well done: 

(1) OO concepts applied
(2) Updated user guide
(3) Provided sufficient test cases. 

You may close this PR after reviewing.

**Pull request closed without merging**